qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1344,6 +1346,8 @@ def jobs(
                     "The 'instance' keyword is only supported for ``ibm_quantum`` runtime."
                 )
             hub, group, project = from_instance_format(instance)
+        if job_tags:
+            validate_job_tags(job_tags, IBMBackendValueError)
```

Comment:
```suggestion
            validate_job_tags(job_tags, ValueError)
```

In fact we can probably just hard code `ValueError` in `validate_job_tags`. I don't remember why it was done this way originally.

Reply:
Removed the `exception` parameter. Use `validate_job_tags` from this repo instead of from the provider.